### ISSUE TITLE
Compilation error in documentation example for Core > Resources

### ISSUE THREAD

**Issue by chris-orchard** [dropwizard/dropwizard#3124]
I don't think the code example in the manual that was using a `LongParam` and `IntParam` in the [1.3.x branch here](https://github.com/dropwizard/dropwizard/blob/release/1.3.x/docs/source/manual/core.rst#resources), but has been updated to a `OptionalLong` and `OptionalInt` in the [2.0 branch here](https://github.com/dropwizard/dropwizard/blob/release/2.0.x/docs/source/manual/core.rst#resources) will compile anymore.

In the `fetch` and `add` methods there are still calls to `userId.get()` and `count.get()` but these are methods that don't exist on the new optional parameter types, only in the previous `LongParam` and `IntParam` types.

**Comment by joschi** [comment]
@chris-orchard Good catch and thanks for reporting this!

Would you be able to create a PR for the documentation to use the `OptionalInt` and `OptionalLong` variants?

**Comment by chris-orchard** [comment]
The reason I found this is that I've got some similar code in my own application that isn't working as expected. I thought I'd check the docs and found they had the same issue I'm facing in #3123. I'm not currently able to get `@DefaultValue` working for `OptionalLong`. Once I've got a solution to that problem I might also be able to correct these docs but for now I'm unsure what the correct code should be for the example,
